tests/topotests/static_routing_with_ebgp/test_static_routes_topo2_ebgp.py

```diff
@@ -0,0 +1,2338 @@
+        "Error: Routes are still present in RIB".format(tc_name)
+
+    step("Random no shut of the nexthop interfaces")
+    for addr_type in ADDR_TYPES:
+        intf = topo["routers"]["r2"]["links"]["r1-link" + str(randnum)]["interface"]
+        shutdown_bringup_interface(tgen, dut, intf, True)
+        nhip = NEXT_HOP_IP["nh" + str(randnum + 1)][addr_type]
+        result = verify_rib(
+            tgen, addr_type, dut, input_dict_5, next_hop=nhip, protocol=protocol
+        )
+        assert result is True, "Testcase {} : Failed \n"
+        "Error: Routes are missing in RIB".format(tc_name)
+
+    dut = "r2"
+    step("Shut the interfaces connected between R2 and R3 one by one")
+    for nhp in range(0, 3):
+        intf = topo["routers"]["r2"]["links"]["r3-link" + str(nhp)]["interface"]
+        shutdown_bringup_interface(tgen, dut, intf, False)
+
+    protocol = "bgp"
+    for addr_type in ADDR_TYPES:
+        input_dict_4 = {"r2": {"static_routes": [{"network": PREFIX1[addr_type]}]}}
+        result = verify_rib(
+            tgen, addr_type, dut, input_dict_4, protocol=protocol, expected=False
+        )
+        assert result is not True, "Testcase {} : Failed \n"
+        "Error: Routes are still present in RIB".format(tc_name)
+
+    step("No shut the interfaces between R2 and R3 one by one")
+    dut = "r2"
+    for nhp in range(0, 3):
+        intf = topo["routers"]["r2"]["links"]["r3-link" + str(nhp)]["interface"]
+        shutdown_bringup_interface(tgen, dut, intf, True)
+    step(
+        "After each interface shut and no shut between R2 -R3 ,verify static"
+        "route is present in the RIB & FIB of R3 & R2 RIB/FIB is remain"
+        " unchanged"
+    )
+    protocol = "static"
+    for addr_type in ADDR_TYPES:
+        input_dict_4 = {"r2": {"static_routes": [{"network": PREFIX1[addr_type]}]}}
+        result = verify_rib(tgen, addr_type, dut, input_dict_4, protocol=protocol)
+        assert result is True, "Testcase {} : Failed \n"
+        "Error: Routes are missing in RIB".format(tc_name)
+
+    protocol = "static"
+    dut = "r2"
+    for addr_type in ADDR_TYPES:
+        input_dict_4 = {"r2": {"static_routes": [{"network": PREFIX1[addr_type]}]}}
+        result = verify_rib(tgen, addr_type, dut, input_dict_4, protocol=protocol)
+        assert (
+            result is True
+        ), "Testcase {}: Failed \n " "Error: Routes are missing in RIB".format(tc_name)
+
+    protocol = "bgp"
+    dut = "r3"
+    for addr_type in ADDR_TYPES:
+        input_dict_4 = {"r2": {"static_routes": [{"network": PREFIX1[addr_type]}]}}
+        result = verify_rib(tgen, addr_type, dut, input_dict_4, protocol=protocol)
+        assert (
+            result is True
+        ), "Testcase {}: Failed \n " "Error: Routes are missing in RIB".format(tc_name)
+
+    step("Reload the FRR router")
+    # stop/start -> restart FRR router and verify
+    stop_router(tgen, "r2")
+    sleep(5)
```

Comment:
here too

Reply:
Changes made are:
1. Removed the static sleep.
2. Base config is modified to use bgp aggressive timers.